packages/sdk-scripts/src/loaders/loadService.ts

```
@@ -4,12 +4,26 @@ import yaml from "js-yaml";
 import { Context } from "../lib/internal";
 import { apiDir } from "./env";
 import { Api } from "../lib/internal";
-import { expectDocVersion } from "../utils";
+import { isObject } from "lodash";
+import { expectDocVersion, extractProviderContract } from "../utils";
+import { i18nYamlFile } from "./env";
 
 export function loadService(serviceSeg: string): Context {
   const context = new Context(serviceSeg);
 
   const serviceDir = path.join(apiDir, serviceSeg);
+
+  const serviceI18nPath = path.join(serviceDir, i18nYamlFile);
+  if (fs.existsSync(serviceI18nPath)) {
+    const i18nData = yaml.safeLoad(fs.readFileSync(serviceI18nPath, "utf8"));
+    // istanbul ignore else
+    if (isObject(i18nData)) {
```

这里是不是不要判断了，有问题直接报错，Fail Fast 原则。